tests/forms_tests/tests/test_validators.py

```diff
@@ -1,9 +1,17 @@
 import re
+import types
 from unittest import TestCase
 
 from django import forms
 from django.core import validators
 from django.core.exceptions import ValidationError
+from django.core.validators import (
+    MaxLengthValidator, MaxValueValidator, MinLengthValidator,
+    MinValueValidator, RegexValidator, URLValidator, int_list_validator,
+    validate_comma_separated_integer_list, validate_email, validate_integer,
+    validate_ipv4_address, validate_ipv6_address, validate_ipv46_address,
+    validate_slug, validate_unicode_slug,
+)
```

Comment:
I think we can chop import and use e.g. `validators.MaxLengthValidator` like in other tests.